mkdocs.yml

```diff
@@ -58,6 +58,13 @@ nav:
     - Modules:
         - modules/index.md
         - modules/couchbase.md
+        - modules/k3s.md
+        - modules/k3s.md
+        - modules/k3s.md
+        - modules/k3s.md
+        - modules/k3s.md
+        - modules/k3s.md
+        - modules/k3s.md
```

Comment:
Only one entry of the k3s module should exist. Probably coming from multiple runs of the code generation tool
```suggestion
```

Reply:
Is this by design that happens while generation or we can take it as an improvement for our module generator?

Reply:
I think an improvement, preferably in a follow up PR